### DISCUSSION TITLE
Change background colour when using SSH?

### DISCUSSION THREAD

**Discussion by xircon** [ghostty-org/ghostty#4068] [Feature Requests, Ideas] [answered]
Equivalent to the kitty ssh kitten.   Is this possible?  Or can I change the background on the fly?

**Comment by xircon** [comment, marked as answer]
OK,  workaround, opens in a new terminal window, but works:
```
#!/usr/bin/env bash

ghostty --background=330055 -e "ssh htpc"
```
Obviously replace the command in the quotes!

**Reply by rhodes-b** [reply]
#2353 for programmatic control of ghostty for the future
